queries/superhtml/highlights.scm

```diff
@@ -0,0 +1,44 @@
+(doctype) @constant
+
+(comment) @comment
+
+(tag_name) @tag
+
+((tag_name) @string.special
+  (#any-of? @string.special "super" "extend"))
+
+(attribute_name) @attribute
+
+(attribute_value) @string
+
+((element
+  (start_tag
+    (attribute
+      (attribute_name) @_attribute
+      [
+        (attribute_value) @markup.link.url
+        (quoted_attribute_value
+          (attribute_value) @markup.link.url)
+      ]))
+  (element
+    (start_tag
+      (tag_name) @tag)))
+  (#eq? @tag "super")
+  (#eq? @_attribute "id"))
```

Comment:
This is risky. Are you wanting to capture the tag_name and its corresponding attributes, or capturing elements whose have `id` attributes and have child `super` elements?

For example, with an HTML code of


```
<foo>
  <bar id="something">
  <bar1>
  <baz>
  <bez>
  <super>
</foo>
```
would super be highlighted as `@tag` here?

Reply:
Hmm I'm not 100% positive the original intent of the author, but based on usage of SuperHTML I believe it makes sense to want to capture elements who have `id` and a `<super>` child.

In your example, I believe `<super>` should be highlighted as `@tag`

Reply:
Yes, can confirm, it in this example super and the id attribute should be highlighted

Reply:
Can you temporarily comment this query out? I'm afraid this is too costly, considering this will capture all non top-level `element` nodes (which there is guaranteed to be a lot of them) before filtering.